lib/ExpensiMark.js

```diff
@@ -54,6 +58,8 @@ export default class ExpensiMark {
                 name: 'link',
                 regex: new RegExp(`\\[([\\w\\s\\d!?&#;]+)\\]\\(${URL_REGEX}\\)(?![^<]*(<\\/pre>|<\\/code>))`, 'gi'),
```

Comment:
It feels weird to have both "regex" and "process" defined here, yet are only calling the process. I would recommend either moving the regex definition into the function, or into the arrow function in "process".